Controller/Adminhtml/Post/Show.php

```diff
@@ -1,15 +1,13 @@
 <?php
 
+declare(strict_types=1);
 namespace Mastering\SampleModule\Controller\Adminhtml\Post;
 
 use Magento\Backend\App\Action;
 use Magento\Framework\Controller\ResultFactory;
 
 class Show extends Action
 {
-    /**
```

Comment:
why you removed the code block?

Reply:
i didnt remove these code blocks
but because i didnt understand why it happend for me
i wanted to commit these file again to win branch
but when i try to commit them form the terminal . it said to me :
no changes added to commit
you know why it say this to me?!